project/build.scala

```diff
@@ -80,19 +88,22 @@ object BananaRdfBuild extends Build {
   lazy val ntriplesM  = CrossModule(crossBuildType,
     id                = "ntriples",
     baseDir           = "io/ntriples",
-    defaultSettings   = buildSettings,
+    defaultSettings   = buildSettings++ Seq(aggregate in Test := false),
```

Comment:
@InTheNow , maybe you can explain? Everything works fine for me even when I delete "aggregate in Test" in all places